crates/turborepo-auth/Cargo.toml

```diff
@@ -5,9 +5,10 @@ edition = "2021"
 license = "MPL-2.0"
 
 [dependencies]
-anyhow.workspace = true
+anyhow = { workspace = true, features = ["backtrace"] }
```

Comment:
Made this match how it's specified in `turborepo-lib`

Reply:
Not that it is blocking in anyway, but Cargo resolves features in an additive way so when `turborepo-lib` is built any of it's dependencies will get a version of `anyhow` with the `backtrace` feature. [Feature unification](https://doc.rust-lang.org/cargo/reference/features.html#feature-unification)